openadapt/thirdparty_customization/my_queue.py

```diff
@@ -0,0 +1,92 @@
+from multiprocessing.queues import Queue
```

Comment:
```suggestion
# retrieved from https://gist.github.com/FanchenBao/d8577599c46eab1238a81857bb7277c9
from multiprocessing.queues import Queue
```
I think it'd be nice to give credit

Reply:
💯

Reply:
[a612535](https://github.com/MLDSAI/OpenAdapt/pull/318/commits/a612535aea4f281c431f94f555dadfecfdf3b941)